index.js

```diff
@@ -123,17 +124,24 @@ module.exports = function (host_or_multiaddr, port) {
     var file
 
     for (var i = 0; i < files.length; i++) {
+
       file = files[i]
-      if (file instanceof stream.Stream || Buffer.isBuffer(file)) {
+
+      if (Buffer.isBuffer(file)) {
+        file = new File({
+          cwd: '/',
+          base: '/',
+          path: '/',
+          contents: streamifier.createReadStream(file)
+        })
+      } else if (file instanceof stream.Stream) {
         file = new File({
           cwd: '/',
           base: '/',
           path: '/',
           contents: file
         })
-      }
```

Comment:
actually no, you want this to be a separate part altogether. the `} else if {` breaks it i thik